Engine/Code/Engine/Graphics/Vertex.cpp

```diff
@@ -0,0 +1,33 @@
+﻿#include "Vertex.h"
+
+std::vector<VertexBufferElement> VertexPCU::m_Attributes = {
+	VertexBufferElement(GL_FLOAT, 3, GL_FALSE),
```

Comment:
You shouldn't be using GL types for this : Make your own abstraction of these values and use those .. This avoid the need to include GL stuff everywhere...